flake.nix

```diff
@@ -99,28 +99,7 @@
         package = { dependencyOverrides ? { }, ... }@args:
           pkgs.callPackage self
           (args // { dependencyOverrides = (inputs // dependencyOverrides); });
-      }) // eachSystem [ "x86_64-linux" "aarch64-darwin" ] (system: {
```

Comment:
From what I understand in the past this used to be like this because we ran `nix flake checks` to do the testing, but we eventually switch to `nix build .#checks.x86_64-linux.init-example-el` hardcoded in the test code, so it shouldn't be an issue to expose checks for all default systems now.